dm/dm/worker/subtask.go

```diff
@@ -515,7 +559,7 @@ func (st *SubTask) Pause() error {
 // TODO: similar to Run, refactor later.
 func (st *SubTask) Resume(relay relay.Process) error {
 	if !st.initialized.Load() {
-		st.Run(pb.Stage_Running, relay)
+		st.Run(pb.Stage_Running, pb.Stage_Running, relay)
```

Comment:
not sure user want to start validator when resume task @sunzhaoyang

Reply:
changed to get validator stage from etcd.